src/foundry/client-esm/data/region-behaviors/display-scrolling-text.d.mts

```diff
@@ -1,34 +1,46 @@
 import type RegionBehaviorType from "./base.d.mts";
-import type { ValueOf } from "../../../../utils/index.d.mts";
+import type { Brand } from "../../../../utils/index.d.mts";
 import fields = foundry.data.fields;
+import type { InvertObject } from "../../../common/utils/helpers.d.mts";
 
 declare namespace DisplayScrollingTextRegionBehaviorType {
-  const _VISIBILITY_MODES: Readonly<{
-    /**
-     * Display only for gamemaster users
-     */
-    GAMEMASTER: 0,
-
-    /**
-     * Display only for users with observer permissions on the triggering token (and for the GM)
-     */
-    OBSERVER: 1,
-
-    /**
-     * Display for all users
-     */
-    ANYONE: 2,
-  }>
-  type VISIBILITY_MODES = ValueOf<typeof _VISIBILITY_MODES>;
+  type _VISIBILITY_MODES = Brand<number, "DisplayScrollingTextRegionBehaviorType.Modes">;
```

Comment:
Nit; name `VISIBILITY_MODES`

Also this should be branded as `DisplayScrollingTextRegionBehaviorType.VISIBILITY_MODES`